requirements/edx/base.txt

```diff
@@ -91,6 +91,7 @@ django-ratelimit-backend==0.6
 unicodecsv==0.9.4
 django-require==1.0.6
 pyuca==1.1
+wrapt==1.10.5
```

Comment:
There are instructions at the top of this file related to adding new
dependencies; please follow them. Thanks!

On Wed, Jul 8, 2015 at 10:07 PM, Utkarsh <notifications@github.com> wrote:

> In requirements/edx/base.txt
> <https://github.com/edx/edx-platform/pull/8835#discussion_r34218229>:
>
> > @@ -91,6 +91,7 @@ django-ratelimit-backend==0.6
> >  unicodecsv==0.9.4
> >  django-require==1.0.6
> >  pyuca==1.1
> > +wrapt==1.10.5
>
> Do we need to add this new dependency? Or is there something we've already
> installed that might suffice?
>
>    - Most probably yes, I tried using functools.wraps, but it does not
>    works in desired way, specially when it comes to decorating functions with
>    arguments, and retaining their identity. Advantage of using wrapt is
>    that it is more extensive, and handles argument specs very well. I believe
>    that it would prove useful in future too.
>
> Have you checked with the open source team about this?
>
>    - Not yet.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/8835/files#r34218229>.
>